flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/DorisTableConfig.java

```diff
@@ -0,0 +1,76 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.flink.tools.cdc;
+
+import org.apache.flink.annotation.VisibleForTesting;
+
+import org.apache.commons.collections.MapUtils;
+
+import java.io.Serializable;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class DorisTableConfig implements Serializable {
+    private static final String LIGHT_SCHEMA_CHANGE = "light_schema_change";
+    // PROPERTIES parameter in doris table creation statement. such as: replication_num=1.
+    private final Map<String, String> tableProperties;
+    // The specific parameters extracted from --table-conf need to be parsed and integrated into the
+    // doris table creation statement. such as: table-buckets="tbl1:10,tbl2:20,a.*:30,b.*:40,.*:50".
+    private Map<String, Integer> tableBuckets;
+
+    public DorisTableConfig(Map<String, String> tableConfig) {
+        if (MapUtils.isNotEmpty(tableConfig)) {
```

Comment:
If tableConfig is null, do we need to assign tableProperties to a new hashmap?

Reply:
sure